src/ads/ads-controller.js

```diff
@@ -34,6 +39,38 @@ class AdsController extends FakeEventTarget implements IAdsController {
     return this._allAdsCompleted;
   }
 
+  /**
+   * @public
+   * @returns {boolean} - Whether we're in an ad break.
+   */
+  adBreak(): boolean {
```

Comment:
if this is a boolean then naming convention should indicate it `isInAdBreak` or `isAdBreak`

Reply:
done